plugins/core/hu.bme.mit.gamma.querygenerator.gui/src/hu/bme/mit/gamma/querygenerator/application/View.java

```diff
@@ -10,6 +10,30 @@
  ********************************************************************************/
 package hu.bme.mit.gamma.querygenerator.application;
 
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.HASHTABLE_SIZE_1024;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.HASHTABLE_SIZE_256;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.HASHTABLE_SIZE_512;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.HASHTABLE_SIZE_64;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.HASHTABLE_SIZE_DEFAULT;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.SEARCH_ORDER_BF;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.SEARCH_ORDER_DEFAULT;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.SEARCH_ORDER_DF;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.SEARCH_ORDER_OF;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.SEARCH_ORDER_RDF;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.SEARCH_ORDER_RODF;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REDUCTION_AGGRESSIVE;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REDUCTION_CONSERVATIVE;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REDUCTION_DEFAULT;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REDUCTION_NONE;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REPRESENTATION_DBM;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REPRESENTATION_DEFAULT;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REPRESENTATION_OA;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REPRESENTATION_UA;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.TRACE_DEFAULT;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.TRACE_FASTEST;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.TRACE_SHORTEST;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.TRACE_SOME;
```

Comment:
If every setting from UppaalSettings is used here, maybe we could consider using `import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.*;` instead.

Reply:
Eclipse replaced it automatically.